june dev update (#107)
irods/irods.org: content/posts/irods-development-update-june-2025.md

Title: iRODS Development Update: June 2025
Date: 2025-06-30 10:00
Author: Kory Draughn
Slug: irods-development-update-june-2025
Status: published

Hello reader,

The 17th Annual iRODS User Group Meeting was a success! As always it was a pleasure speaking with members of the community and listening to the interesting things they are working on.

> For those who could not attend, you'll be happy to know that presentations are now available on the [iRODS Consortium YouTube channel](https://youtube.com/playlist?list=PL29FhEN41mZNGY9cCOoukDDx4xpF4HN0v&si=3EL1FcvCN9geb-AE).

Reviewer: ```suggestion
For those who could not attend, you'll be happy to know that abstracts, slides, and video are now available from [https://irods.org/ugm2025](https://irods.org/ugm2025).
```
